CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Motoko `bitcoin` changelog
+
+## 0.1.1
```

Comment:
Should `version` in `mops.toml` also be updated to `0.1.1`?

Reply:
Yes, done